experimental/framework/get-container/src/routerliciousService.ts

```diff
@@ -74,4 +80,23 @@ export class RouterliciousService implements IGetContainerService {
 
         this.urlResolver = new SimpleUrlResolver(config, user);
     }
+
+    public async createContainer(
```

Comment:
Who's using these calls?

Reply:
These were added since routerliciousService needed to meet the requirements of the updated IGetContainerService interface. I've removed all changes to get-container's logic now since tinylicious-client uses its own implementation